docs/main/ios/custom-code.md

```diff
@@ -29,7 +29,13 @@ Copy the following Swift code into `EchoPlugin.swift`:
 import Capacitor
 
 @objc(EchoPlugin)
```

Comment:
You could actually remove this line since the EchoPlugin.m file is removed, the only time `@objc(ClassName)` is needed is if the class needs to be initialized in objective-c directly.

Reply:
Interesting, but we have not removed that from any of the core plugins, so I think I'll leave t here for consistency